.github/workflows/build.yml

```diff
@@ -0,0 +1,90 @@
+name: Build and push bento_beacon
+# -- TEMP:
+# on:
+#   release:
+#     types: [ published ]
```

Comment:
production versions won't get the semver / latest flag on the container unless the release event triggers the workflow